### DISCUSSION TITLE
Why is `@babel/runtime-corejs3` a dependency?

### DISCUSSION THREAD

**Discussion by yuri-constante-thescore** [uNmAnNeR/imaskjs#884] [Q&A]
When updating to imask 7 I saw that `@babel/runtime-corejs3` was added as a dependency, but I couldn't find a usage of it or its dependencies. 

```
> grep -rE 'core-js|corejs|regenerator' 'node_modules/imask'
node_modules/imask/package.json:    "@babel/runtime-corejs3": "^7.22.5"
```

**Comment by uNmAnNeR** [comment]
hi! that's true that it's not used at the moment. I added it as suggested in the documentation for babel just to make sure that all features are compiled when i add something in the future.

**Comment by yuri-constante-thescore** [comment]
Is it possible to configure Babel to never create code that depend on this dependency?
I'm working on a project that doesn't use babel, so this change added 16Mb to our node_modules.

**Reply by uNmAnNeR** [reply]
that's a purpose of babel to make the new code to work in the old environments which means compiling + injecting polyfills.

So for example if sometime i will use new stuff like `private` fields etc. then the babel will inject required polyfills. So i can not worry about the features i use. Profit.

but why is this a problem for you? Unused modules are dropped by a bundler anyway.

**Reply by yuri-constante-thescore** [reply]
Just trying to keep a small node_modules, our project had a node_modules of 25Mb, so this change was a significant increase.

**Reply by uNmAnNeR** [reply]
Well personally I prefer not to worry about compilation rather than counting megabytes. But i don't mind removing it completely if you have a way to remind me if it needs to be included sometime later.  Should be some test or something
